scaling_performance/network_optimization.adoc

```diff
@@ -137,4 +137,20 @@ osm_cluster_network_cidr=10.128.0.0/10
 
 This will allow for 8192 nodes, each with 510 usable IPs.
 
-Please refer to the supportability limits in the {product-title} documentation for node/pod limits for the version of software you are installing.
+See the supportability limits in the {product-title} documentation for node/pod limits for the version of software you are installing.
+
+[[scaling-performance-optimizing-ipsec]]
+== Optimizing IPSec
+
+Because encrypting and decrypting node hosts uses CPU power, performance will be
+affected both in throughput and CPU usage on the nodes when encryption is
+enabled, no matter what IP security system is used.
```

Comment:
s/no matter what IP security system is used/regardless of the IP security system being used
"no matter what..." seems a bit slangy.